parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -503,13 +531,26 @@ uncons :: Applicative m => Branch m -> m (Maybe (Branch0 m, Branch m))
 uncons (Branch b) = go <$> Causal.uncons b where
   go = over (_Just . _2) Branch
 
-stepManyAt :: (Monad m, Foldable f)
-           => f (Path, Branch0 m -> Branch0 m) -> Branch m -> Branch m
-stepManyAt actions = step (stepManyAt0 actions)
+-- | Run a series of updates at specific locations efficiently, aggregating all changes
+-- into a single causal step.
```

Comment:
What does "efficiently" refer to in the title and various comments?  Does it just mean aggregated into a single causal step?